iChef-WebUi/src/app/components/recipe-page/recipe-page.component.html

```diff
@@ -0,0 +1,128 @@
+<body>
+  <h1>Recipe Creation</h1>
+  <div class="left">
+   
+      <div class="form-field">
+        <input class="input" text-align="center" placeholder="Recipe Title">
+      </div>
+
+
+   <br>      
+          &nbsp;
+        <label class="form-field-smaller">
+          Ingredients: </label>&nbsp;&nbsp;
+        
+        <span class="form-field-smaller">
+          <input class="input-smaller" >
+        </span>&nbsp;&nbsp;
+        
+        <span class="form-field-smaller">
+          <input class="input-smaller">
+        </span>&nbsp;&nbsp;
+
+        <!-- <br> -->
```

Comment:
Please clear the commented lines from your code. Keep only the explanation comments.